presto-spi/src/main/java/com/facebook/presto/spi/statistics/TableStatistics.java

```diff
@@ -111,6 +121,7 @@ public static final class Builder
         private Estimate rowCount = Estimate.unknown();
         private Estimate totalSize = Estimate.unknown();
         private Map<ColumnHandle, ColumnStatistics> columnStatisticsMap = new LinkedHashMap<>();
+        private ConfidenceLevel confidenceLevel = FACT;
```

Comment:
1. @kaikalur pointed out that table scan statistics aren't facts like enforce single row is.  HIGH would be more appropriate.
2. I wonder if LOW is a better default, and then explicitly set something different when you know the confidence is higher (and probably most tablestatistics would set this to high).  I'm a bit torn about what the right default is here. @kaikalur what do you think?